### ISSUE TITLE
Add option to disable "TOP" videos (for better privacy on small instances)

### ISSUE THREAD

**Issue by Perflyst** [omarroth/invidious#294]
Add option to disable "TOP" videos (for better privacy on small instances)

If there are only some users on a new invidious instance it leaks interest of them to the public.

**Comment by omarroth** [comment]
Several issues all appear to be clustered around allowing administrators to configure various aspects of the site, such as login, feeds, and homepage. 

This is also related to allowing authors to customize comments and potentially other channel preferences (see some of the conversation in #264).

I'm not sure how I'd like to handle this currently. Admin options could be configured from the `config.yml`, but it would be better IMO to have "roles" so most code could be reused for allowing authors similar functionality.

If it should be better to manage preferences through the site itself, rather than through a config, I'll open a new issue to track it and close #285, #293, #294, and #295.

**Comment by omarroth** [comment]
Rather than closing the mentioned issues, I opened a new one to help track them. I expect they will be added fairly quickly after #312 is implemented.

**Comment by omarroth** [comment]
Added with a39b1583dacff31664830fd4f8966014f8869fa0.
